src/App.js

```diff
@@ -17,36 +17,35 @@ class App extends Component {
   }
 
   render() {
+    const { data: { user } } = this.props
     return (
       <div className="App">
         <div className="App-header">
           <h2>Code for Denver Members</h2>
-          <Login user={this.props.data.user} />
+          <Login user={user} />
+          <Link to="/user-list">All Users</Link>
         </div>
         <Route exact path="/" component={MemberResources} />
         <Route
           exact path="/me"
-          render={() =>
-            <MemberProfile
-              user={this.props.data.user}
-            />}
+          render={() => <MemberProfile user={user} />}
         />
         <Route path="/me/edit" component={MemberProfileEdit} />
-      </div>
+        <Route path="/user-list" component={UsersList} />      </div>
```

Comment:
For this story, being a list of emails for the admin to add people, I think the route should be something like `/admin/onboarding`, and then card #55 could be the user list that would display more public info, like name, etc.

Reply:
That sounds good to me, I'll make those changes now.